### PR TITLE
Consistency in the restart policy docs

### PR DESCRIPTION
**Pull request by moxiegirl**
Follow onto #11235 final consistency

Signed-off-by: Mary Anthony mary.anthony@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
@jamtur01 @fredlf @SvenDowideit

File: docs/man/docker-create.1.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ This value should always larger than **-m**, so you should alway use this with *
    Mount the container's root filesystem as read only.
 
 **--restart**="no"
-   Restart policy to apply when a container exits (no, on-failure[:max-retry], always)
+   Restart policy when a container exits (no, on-failure[:max-retry], always)
```
Comment (jamtur01) [inline docs/man/docker-create.1.md]: This doesn't make sense to me - for when?

**Comment by ahmetalpbalkan** [comment]
typo in the commit message, not sure worth fixing.

File: docs/man/docker-create.1.md (hunk 1 above)
Comment (ahmetalpbalkan) [inline docs/man/docker-create.1.md]: "Restart policy for an exited container" ?

**Comment by moxiegirl** [comment]
@ahmetalpbalkan @jamtur01 This one has already gone rounds on another PR.  I'm just tweaking the text cause non-native speaker.

**Comment by SvenDowideit** [comment]
LGTM @docker/docs-owners

**Pull request merged**